### PR TITLE
AG-10089 Handle duplicate series ID by ignored duplicates

### PR DESCRIPTION
**Pull request by olegat**
https://ag-grid.atlassian.net/browse/AG-10089

### CODE REVIEW

File: packages/ag-charts-community/src/chart/mapping/prepareSeries.ts [hunk 1]
```diff
@@ -89,6 +101,13 @@ export function processSeriesOptions(_opts: AgChartOptions, seriesOptions: Serie
             Logger.warnOnce(`unsupported stacking of series type: ${sType}`);
         }
 
+        // AG-10089 Ignore duplicate user input id values:
+        if (idMatchIndex !== undefined) {
+            Logger.warn(`series[${index}].id "${series.id}" ignored because it duplicates series[${idMatchIndex}].id`);
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/mapping/prepareSeries.ts]: Nit: This should be warnOne() to avoid spamming the console.

**Review by alantreadway — COMMENTED** [review]
LGTM other than one nit.

File: packages/ag-charts-community/src/chart/mapping/prepareSeries.ts (hunk 1 above)
Comment (olegat) [reply packages/ag-charts-community/src/chart/mapping/prepareSeries.ts]: Done.

**Review by alantreadway — APPROVED** [review]
LGTM

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**